janus_client/src/lib.rs

```diff
@@ -202,10 +200,39 @@ where
     }
 }
 
+#[cfg(test)]
+mod test_util {
+    use std::sync::Once;
+    use tracing_log::LogTracer;
+    use tracing_subscriber::{prelude::*, EnvFilter, Registry};
+
+    /// Install a tracing subscriber for use in unit tests. This should be called at the beginning
+    /// of any test that requires a tracing subscriber.
+    pub fn install_test_trace_subscriber() {
```

Comment:
This feels very, very similar to `janus_server::trace::test_util::install_test_trace_subscriber`. Maybe this should be in the `test_util` crate in the `janus` workspace so that both `janus_server` and `janus_client` can use it?

Reply:
It is similar in effect to `janus_server::trace::test_util::install_test_trace_subscriber`, but pared down, without the configurability that `install_trace_subscriber()` has via `TraceConfiguration`. I'll give that move a try -- this could remove one more `#[doc(hidden)]` item in `janus_server`, which is nice.

Reply:
It may not be worth the complexity of further parameterizing `janus_server::trace::test_util::install_test_trace_subscriber`, since the composed types get really gnarly, and since we probably won't have to change these tracing subscribers much, such that the benefit of de-duplicating code may be marginal. Your call.